tests/subsys/emul/prj.conf

```diff
@@ -0,0 +1,12 @@
+# Copyright 2022 Google LLC
+#
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_ZTEST=y
+CONFIG_ZTEST_NEW_API=y
+CONFIG_I2C=y
+CONFIG_SENSOR=y
+CONFIG_BMI160=y
```

Comment:
this is now redundant

Reply:
Removed redundant entries. Removing any of the remaining ones will cause the build to fail.